mendix/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "4119b134-c828-4e14-95b5-585bb13d314a",
+  "app_id": "mendix",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Market leading low-code IDE for application development",
+    "title": "Mendix",
+    "media": [
+    {
+      "media_type": "image",
+      "caption": "Mendix Studio Pro 9",
+      "image_url": "images/mx9_splash_screen.png"
+    },
```

Comment:
```suggestion
```

Please only include images that are relevant to the integration.

Reply:
As we don't have any dashboards yet, I guess this is fine for the first pass. We could revisit this once we have screenshots of relevant DD dashboards

Reply:
Please remove this image

Reply:
I've removed the image.